cuML/src/cuML.hpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <memory>
+#include <vector>
```

Comment:
We don't need this header file, right?

Reply:
The header file is currently not used, but its intention is to contain the public external C++ interface of cuML.

Reply:
Sorry, I meant we are not using `std::vector` in the current header file, so we don't this this '#include', right?

Reply:
Yes. I will remove it.